nixos/modules/programs/xss-lock.nix

```diff
@@ -8,20 +8,37 @@ in
 {
   options.programs.xss-lock = {
     enable = mkEnableOption "xss-lock";
+
     lockerCommand = mkOption {
       default = "${pkgs.i3lock}/bin/i3lock";
       example = literalExample ''''${pkgs.i3lock-fancy}/bin/i3lock-fancy'';
       type = types.string;
       description = "Locker to be used with xsslock";
     };
+
+    extraOptions = mkOption {
```

Comment:
this is named `extraOptions` but there are no other options. I did this because of consistency with `xautolock` but it might make more sense to name it `options`. Some with more experience should decide.

Reply:
`extraOptions` is usually used when adding some special service options in the verbatim format for the software itself (in this case CLI args). The default and most important option is `lockerCommand`, so it's perfectly fine to specify all the remaining options using `extraOptions`. :+1: